src/arrays.js

```diff
@@ -1,61 +1,76 @@
 const getNthElement = (index, array) => {
-  // your code here
+  m = (array.length);
+  if(index>0){
+    index = index % m;
```

Comment:
I like your thinking here. It is good practice not to overwrite a function parameter in your code. However, in this case it does work and we don't need to use the original `index` variable for anything else. I would usually declare a new variable.